src/libs/actions/IOU.js

```diff
@@ -2791,6 +2805,11 @@ function getPayMoneyRequestParams(chatReport, iouReport, recipient, paymentMetho
             key: ONYXKEYS.NVP_LAST_PAYMENT_METHOD,
             value: {[iouReport.policyID]: paymentMethodType},
         },
+        {
+            onyxMethod: Onyx.METHOD.SET,
+            key: `${ONYXKEYS.COLLECTION.NEXT_STEP}${iouReport.reportID}`,
+            value: null,
+        },
```

Comment:
I think ti might be more efficient to only do this when the currentNextStep is not `null`